targets/csharp/templates/PlayFabSettings.cs.ejs

```diff
@@ -24,6 +24,27 @@ namespace PlayFab
         public static string DeveloperSecretKey = null;
         /// <summary> You must set this value for PlayFabSdk to work properly (Found in the Game Manager for your title, at the PlayFab Website) </summary>
         public static string TitleId;
+		/// <summary> Placeholder for the Local Debugging URI. </summary>
+		private static string _debugUri;
+		/// <summary> Must be manually set by user on .NETStandard 1.1 and .NETPortable for local debugging to work properly </summary>
+        public static string DebugUri 
```

Comment:
I'm concerned about this name.
The name implies a much more generic purpose than it actually serves. It's only used by one method, and can't be used by another.
Either we need to fix it so that the URI is shared by any potential CloudScript API call (or relevant thing), or we need to   rename it to match the fact that it's very specific to just one method.

Reply:
How about `ExecuteFunctionLocalUri` and `_executeLocalFunctionUri`?

Reply:
I'm happy with a limited name that you propose.
I'd be slightly more happy with a generic name, if that variable was able to serve a more generic purpose.

Reply:
If we go with ExecuteFunctionLocalUri as the public property the field should be _executeFunctionLocalUri, no?

---
In reply to: [262266567](https://github.com/PlayFab/SDKGenerator/pull/431#discussion_r262266567) [](ancestors = 262266567)